File: src/main/resources/static/js/service/chargeRecordService.js
//服务层
app.service('chargeRecordService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../chargeRecord/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../chargeRecord/findPage.do?page='+page+'&rows='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../chargeRecord/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../chargeRecord/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../chargeRecord/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../chargeRecord/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../chargeRecord/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: src/main/java/group/yunxin/service/ChargeRecordService.java
package group.yunxin.service;

import java.util.List;

import group.yunxin.pojo.TbChargeRecord;
import group.yunxin.vo.PageResult;

/**
 * 服务层接口
 * 
 * @author Administrator
 *
 */
public interface ChargeRecordService
{

	/**
	 * 返回全部列表
	 * 
	 * @return
	 */
	public List<TbChargeRecord> findAll();

	/**
	 * 返回分页列表
	 * 
	 * @return
	 */
	public PageResult findPage(int pageNum, int pageSize);

	/**
	 * 增加
	 */
	public void add(TbChargeRecord chargeRecord);

	/**
	 * 修改
	 */
	public void update(TbChargeRecord chargeRecord);

	/**
	 * 根据ID获取实体
	 * 
	 * @param id
	 * @return
	 */
	public TbChargeRecord findOne(Long id);

	/**
	 * 批量删除
	 * 
	 * @param ids
	 */
	public void delete(Long[] ids);

	/**
	 * 分页
	 * 
	 * @param pageNum  当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageResult findPage(TbChargeRecord chargeRecord, int pageNum, int pageSize);

	/**
	 * 得到14天充值记录
	 * @return
	 */
	public List<Double> getRecord();

}
